packages/camera/camera_android/lib/src/utils.dart

```diff
@@ -29,37 +31,91 @@ DeviceOrientation deviceOrientationFromPlatform(
         DeviceOrientation.landscapeRight,
     };
 
+/// Converts a [DeviceOrientation] to [PlatformDeviceOrientation].
+PlatformDeviceOrientation deviceOrientationToPlatform(
+        DeviceOrientation orientation) =>
+    switch (orientation) {
+      DeviceOrientation.portraitUp => PlatformDeviceOrientation.portraitUp,
+      DeviceOrientation.portraitDown => PlatformDeviceOrientation.portraitDown,
+      DeviceOrientation.landscapeLeft =>
+        PlatformDeviceOrientation.landscapeLeft,
+      DeviceOrientation.landscapeRight =>
+        PlatformDeviceOrientation.landscapeRight,
+    };
+
 /// Converts a [PlatformExposureMode] to [ExposureMode].
 ExposureMode exposureModeFromPlatform(PlatformExposureMode exposureMode) =>
     switch (exposureMode) {
       PlatformExposureMode.auto => ExposureMode.auto,
       PlatformExposureMode.locked => ExposureMode.locked,
     };
 
+/// Converts a [ExposureMode] to [PlatformExposureMode].
+PlatformExposureMode exposureModeToPlatform(ExposureMode exposureMode) =>
+    switch (exposureMode) {
+      ExposureMode.auto => PlatformExposureMode.auto,
+      ExposureMode.locked => PlatformExposureMode.locked,
+    };
```

Comment:
This needs a failsafe handler too since it's from the platform interface package.